README.md

```diff
@@ -23,26 +23,26 @@ an update is found, the automation creates a pull request that a human user
 can then review and merge. See [`autoupdate.yaml`](#autoupdateyaml) for
```

Comment:
I guess the reference to "images" earlier in this paragraph will be updated when you do the autoupdate PR?